torch/nested/_internal/ops.py

```diff
@@ -880,3 +880,36 @@ def embedding_default(func, *args, **kwargs):
     return NestedTensor(
         func(weight, indices._values, **new_kwargs), **extract_kwargs(indices)
     )
+
+@register_jagged_func(
+    torch.ops.aten._nested_zeros.default,
```

Comment:
> how do we usually deal with overriding factory functions for subclasses? this might be nonsense, but would a NestedTensorMode help us?

This is an interesting approach, though the main downside I think is that the user would have to manually enable the mode. 

> maybe we can use dummy.new_empty() / dummy.new_zeros() / etc. ?

Good idea, I'm going to try this approach.

Reply:
Updated to call into new_{empty,zeros,...}